l10n_it_fatturapa_out_sale/views/partner_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="view_partner_form_fatturapa">
+            <field name="name">partner.form.fatturapa.sale.order.data</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="l10n_it_fatturapa.view_partner_form_fatturapa"/>
+            <field name="arch" type="xml">
+                <field name="electronic_invoice_subjected" position="after">
+                    <field name="fatturapa_sale_order_data"/>
```

Comment:
Questo campo non dovrebbe andare dentro il `group` visibile solo se `electronic_invoice_subjected`?

Reply:
in pratica verrebbe così
`<field name="ipa_code" position="before">`
￼